claim/netdata-claim.sh.in

```diff
@@ -316,32 +338,34 @@ if [ "${HTTP_STATUS_CODE}" = "204" ] ; then
   cloud base url = $URL_BASE
 HERE_DOC
         cp "$T" "$CLAIMING_DIR/cloud.conf"
+        if [ "$EUID" == "0" ]; then
```

Comment:
The temp file is redundant now, we are no longer rewriting the main config. The HERE_DOC can be used to overwrite cloud.conf directly.

Reply:
Yes, I also kept the `sed` command in comment. I didn't want to get rid of the code immediately. Do you think I should clean it up? In the future we might need to update `cloud.conf` selectively, instead of recreating it like now.

Reply:
It will be the history if we need it. Kill it.

Reply:
With fire.